Graphs/testHelper.js

```diff
@@ -0,0 +1,44 @@
+import fsmo from 'fs';
+const cheerio = require('cheerio');
+
+
+export const readJSON = (JSONFile) => {
+    try {
+        return JSON.parse(fsmo.readFileSync(JSONFile, 'utf8'));
+    } catch (e) {
+        console.log(e);
+    }
+}
+
+export const CheckTicks = (component, parentClass, childClass, selector) => {
+    const middleHtml = component.find(parentClass).find(childClass).html();
+    const $ = cheerio.load(middleHtml);
+    const xAxisTicks = $(childClass).find(selector).length;
+    return xAxisTicks;
+}
+
+const readdirAsync = (path) => {
+    return new Promise(function (resolve, reject) {
+        fsmo.readdir(path, function (error, result) {
+            if (error) {
+                reject(error);
+            } else {
+                resolve(result);
+            }
+        });
+    });
+}
+
+export const getDataAndConfig = (graphName) => {
+    return new Promise(async function (resolve, reject) {
+        const dataFolder = __dirname + '/' + graphName + '/mock/data';
+        const configFolder = __dirname + '/' + graphName + '/mock/configurations';
+        let allFiles = await readdirAsync(configFolder);
+        let config = [];
+        allFiles.forEach(file => {
+            config[file.split(".")[0]] = readJSON(configFolder + '/' + file);
+        });
+        config['data'] = readJSON(dataFolder + '/data.json');
+        resolve(config);
+    });
+}
```

Comment:
@mayank-nx 
- Comparison like Bar Width and Height is missing.
- Comparison like positions of the g is missing, that will help us to know that Bar is plotted at exact location.
- Compare it for two random bars.

Reply:
I have created a test case in which I have covered all the above points.